main.yml

```diff
@@ -182,7 +275,7 @@ openshift:
     paths:
       - /openshift
     sub_apps:
-      - id: clusters
```

Comment:
Does this not need an ID?

Reply:
They wanted to configure it so that there was no path parameter for Clusters (so it's just /openshift). Since Clusters isn't an app defined elsewhere in the config, it's not technically required.

Reply:
Okay, that makes sense!

You may want to change overview on [line 210](https://github.com/RedHatInsights/cloud-services-config/blob/9a90fafd043dcd71f22781570a626513e612721e/main.yml#L210) to make it consistent with [line 128](https://github.com/RedHatInsights/cloud-services-config/blob/9a90fafd043dcd71f22781570a626513e612721e/main.yml#L128) since it also isn't defined on its own, but I don't think it's critical or anything.

Reply:
For Insights, though, Overview appends a path parameter to the end. So while Insights' Overview is at `/insights/overview`, Cost Management's Overview is at `/hybrid/cost-management` (no ID appended).